.ci/doc/templates/print-result-array.tpl.js

```diff
@@ -0,0 +1,31 @@
+// Loads the Kuzzle SDK module and the websocket protocol
+const {
+  Kuzzle,
+  WebSocket
+} = require('kuzzle-sdk');
+
+// Instantiates a Kuzzle client
+const
+  kuzzle = new Kuzzle(
+    new WebSocket('kuzzle', { autoReconnect: false })
+  );
+
+// Adds a listener to detect any connection problems
+kuzzle.on('networkError', error => {
+  console.error(`Network Error: ${error.message}`);
+});
+
+(async () => {
+  let result;
+  try {
+    await kuzzle.connect();
+  } catch (error) {
+    console.log(`Can not connect to Kuzzle: ${error.message}`);
+  }
+  [snippet-code] finally {
+    kuzzle.disconnect();
+  }
+  for (const elem of result.successes) {
```

Comment:
This will only works with arrays returned by some of the `m*` methods. For example it wont works with document:search or document:mDelete.

You don't need to use this template also, you can just print the content of the result in JSON with newlines

Reply:
I think this template could be useful, so I just renamed it `print-result-successes` as it do exactly that. So no confusion for route that wont return a `successes` array.